services/backend/src/core/search/util/fuzzySearch.js

```diff
@@ -38,6 +38,7 @@ async function fuzzySearch(profiles, searchValue) {
       "teams",
       "telephone",
       "tenure",
+      "exFeederText",
```

Comment:
you will also have to display this match in the fuzzy search match list on the result card